### PR TITLE
[Bombastic Perks] Playstyle perks

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Mods "Bombastic Perks adds Playstyle Perks"

#### Purpose of change
Allows more powerful perks as an opt in system,

#### Describe the solution
 relies on #65307 for one of the new perks, but might get paired out.

Added a few variations on playstyle perk availability.

### CODE REVIEW

File: data/mods/BombasticPerks/perks.json [hunk 1]
```diff
@@ -252,5 +252,13 @@
     "points": 0,
     "description": "You've been dissecting critters since high school.  Last night a voice gave you an idea for how to bring them back to life.",
     "category": [ "perk" ]
+  },
+  {
+    "type": "mutation",
+    "id": "perk_empath",
+    "name": { "str": "Empath" },
+    "points": 0,
+    "description": "You've always had a pashion for people, and a deep understanding of their thoughts and feelings.  Now that almost everyone's dead that has to extend to other things too!  When you kill a creature gain some of it's ability temporarily.",
```
Comment (TheShadowFerret) [inline data/mods/BombasticPerks/perks.json:261]:
```suggestion
    "description": "You've always had a passion for people, and a deep understanding of their thoughts and feelings.  Now that almost everyone's dead that has to extend to other things too!  When you kill a creature gain some of its abilities temporarily.",
```

File: data/mods/BombasticPerks/corefiles/welcomemenu.json [hunk 2]
```diff
@@ -66,51 +66,111 @@
...
+      },
+      {
+        "text": "Only 1 playstyle perk ever (default)",
+        "effect": [ { "math": [ "u_playstyle", "=", "1" ] },
+        {
+          "u_add_var": "playstyle_note",
+          "type": "dialogue",
+          "context": "perks",
+          "value": "I can select 1 lifestyle perk."
+        } ],
```
[the remark this answers is not in the document]
Comment (TheShadowFerret) [reply data/mods/BombasticPerks/corefiles/welcomemenu.json:122]: ~~One should be the word since it's <=10~~

**Comment by Rimplestilt** [comment]
So i really don't know much about how the perks mod works yet, but is it possible to have certain perks cost more points? Where you may have to go a couple "levels" to unlock it?

My thoughts are adding a section to perks that allows Traits (like from character creation) to be selected. Traits could be marked individually for inclusion in this, so that it isn't universal. And some traits would cost more points to unlock. You could potentially extend this further by actually just using the point system from enemy kills and making every Perk cost a certain amount.

**Comment by bombasticSlacks** [comment]
> My thoughts are adding a section to perks that allows Traits (like from character creation) to be selected. Traits could be marked individually for inclusion in this, so that it isn't universal. And some traits would cost more points to unlock. You could potentially extend this further by actually just using the point system from enemy kills and making every Perk cost a certain amount. 

Feel free to reuse my perks code for a third party mod, or work on a third party extension to the current perks mod (there is a guide to adding your own perks in third party mods). I have no interest in that idea for the core mod though sorry 😔

It's all new "perks" no traits.

**Pull request merged**